packages/cockroach/extra/dcos-cockroachdb-config-change.service

```diff
@@ -5,10 +5,14 @@ Documentation=https://docs.mesosphere.com
 [Service]
 Type=simple
 User=dcos_cockroach
+PermissionsStartOnly=True
```

Comment:
This seems to be now a deprecated option: https://github.com/NixOS/nixpkgs/issues/53852

It should be replaced with command specific prefix documented in
https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=

It seem that `+` would be right prefix to run `bootstrap` script.

This should be a follow up issue.

Reply:
I will make a follow-up issue and agree that it should not be done here because this is the same as our other service files.

Reply:
Issue created: https://jira.mesosphere.com/browse/DCOS_OSS-5129